wrangler.toml

```diff
@@ -5,6 +5,7 @@ type = "webpack"
 
 account_id = "f8ffb577efdf82b0303f86d7b4ca94a5"
 workers_dev = true
+node_compat = true
```

Comment:
What's this?

Reply:
Sorry - just took another look and looks like I didn't run `npm install` so `buffer` wasn't being found. Originally it was refusing to build unless I also added the line. Will remove from the commit!